tests/e2e/pact/services/get-access-token.ts

```diff
@@ -0,0 +1,36 @@
+import {createJWT} from "./jwt"
+import axios from "axios"
+import {URLSearchParams} from "url"
+import * as fs from "fs"
+import path from "path"
+
+/* TODO - get from ADO / env variables */
+const internalDevApiKey = "4PD6IBEDrC4oVlQOJ5VSqk951SAqTA1G"
```

Comment:
is this todo required for this pr?

Reply:
+assuming from the location of this file we're planning on adding a smoke-test in future?

Reply:
yea smoke tests are for a future ticket - felt like this was getting increasingly bloated so cut it off before any smoke test related things happened

Reply:
you can get an access token locally for now and test using that manually? (only in internal-dev)